app/auth/signup/email/page.tsx

```diff
@@ -0,0 +1,113 @@
+'use client'
+import { useRef } from 'react'
+import { useForm } from 'react-hook-form'
+import {
+  Box,
+  Flex,
+  FormLabel,
+  FormErrorMessage,
+  FormControl,
+  Input,
+  Heading,
+  useToast
+} from '@chakra-ui/react'
+import { PrimaryButton } from '@/components/button'
+import { signUp } from './action'
+import { useFormResolver, SignUpSchema } from './schema'
```

Comment:
Please always use alias with absolute path not relative path like `@/auth/signup/email/action`
The frontend rule 
https://github.com/tabi-memo/frontend/wiki/Front-Rules